pepper-apis/src/main/java/org/broadinstitute/ddp/AddressPlusCode.java

```diff
@@ -0,0 +1,32 @@
+package org.broadinstitute.ddp;
+
+import com.typesafe.config.Config;
+import org.broadinstitute.ddp.constants.ConfigFile;
+import org.broadinstitute.ddp.service.OLCService;
+import org.broadinstitute.ddp.util.ConfigManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AddressPlusCode {
```

Comment:
Does this have to live under `src/main`? I'm a bit concerned that folks might get confused and start using it in real code. How about we move to this `src/test`? Maybe just encapsulate this as a `@Test` in `OLCServiceTest` or another test class?

Reply:
It sounds like we just want to run this locally every once in a while right? This should probably do the trick:
```java
public class OLCServiceTest ... {
  ...

  // Uncomment to run this!
  // @Test
  public void computePluscode() {
    ...
  }
}

Reply:
It doesn't need to live under `src/main`, `src/test` sounds good - I was not thinking of that concern, I'll make sure to keep it in mind in the future.

I like the solution proposed! I'll do it that way.